pkg/controllers/oauthclientscontroller/oauthclientscontroller_test.go

```diff
@@ -0,0 +1,532 @@
+package oauthclientscontroller
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"reflect"
+	"testing"
+
+	configv1 "github.com/openshift/api/config/v1"
+	oauthv1 "github.com/openshift/api/oauth/v1"
+	routev1 "github.com/openshift/api/route/v1"
+	configv1listers "github.com/openshift/client-go/config/listers/config/v1"
+	fakeoauthclient "github.com/openshift/client-go/oauth/clientset/versioned/fake"
+	oauthv1listers "github.com/openshift/client-go/oauth/listers/oauth/v1"
+	routev1listers "github.com/openshift/client-go/route/listers/route/v1"
+	"github.com/openshift/library-go/pkg/oauth/oauthdiscovery"
+	"github.com/openshift/library-go/pkg/operator/events"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	clienttesting "k8s.io/client-go/testing"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
```

Comment:
please use https://github.com/openshift-eng/openshift-goimports to organize imports.

Reply:
Updated in 331473e